app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] [REQUIRED] The implementations and functionalities of the devise implementation work fine, however, we need to ensure that the registration and login have been modified to match the wireframes and their styling because this is a hard requirement for this milestone.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="386" alt="Screenshot 2023-07-07 at 00 16 38" src="https://github.com/getssh/blog_app/assets/58692590/afea51ca-236d-408c-bcfe-2bb23252a46b"> | <img width="574" alt="Screenshot 2023-07-07 at 00 17 05" src="https://github.com/getssh/blog_app/assets/58692590/543a00d9-0bf4-46f9-9569-d14ce725055e">